src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -721,15 +716,7 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                 this,
                 diagnostics);
 
-            var ctor = (MethodSymbol)GetWellKnownTypeMember(this.Compilation, TupleTypeSymbol.GetTupleCtor(elements.Length), diagnostics, syntax: node);
-            if ((object)ctor != null)
-            {
-                ctor = ctor.AsMember(type.UnderlyingTupleType);
-            }
-            else
-            {
-                hasErrors = true;
-            }
+            var ctor = type.GetTupleConstructorChain(node, this, diagnostics, ref hasErrors);
```

Comment:
Thanks. I will update `var` in multiple places to follow this convention. Makes sense.